tests/turbo.web.test.ts

```diff
@@ -248,14 +248,51 @@ describe('Browser environment', () => {
         expect(error).to.be.instanceOf(FailedRequestError);
         expect(error.message).to.contain('Failed request: 404: Not Found');
       });
+
+      it('should properly submit an existing payment transaction ID to the Turbo Payment Service for processing a pending tx', async () => {
+        const txId = await sendFundTransaction(1000);
+
+        const { id, winc, owner, token, status } =
+          await turbo.submitFundTransaction({
+            txId,
+          });
+        expect(id).to.equal(txId);
+        expect(owner).to.equal(testWalletAddress);
+        expect(winc).to.equal('766');
+        expect(token).to.equal('arweave');
+        expect(status).to.equal('pending');
+      });
+
+      const minConfirmations = 25;
+      it('should properly submit an existing payment transaction ID to the Turbo Payment Service for processing a confirmed tx', async () => {
+        const balanceBefore = await getRawBalance(testWalletAddress);
+
+        const txId = await sendFundTransaction(1000);
+        for (let i = 0; i < minConfirmations; i++) {
+          await mineArLocalBlock();
+        }
```

Comment:
see comment about `mine` endpoint supporting providing the # of blocks you want to mine - would allow us to remove this